tests/model_serving/model_server/multi_node/conftest.py

```diff
@@ -110,6 +110,58 @@ def multi_node_inference_service(
         yield isvc
 
 
+@pytest.fixture(scope="class")
+def multi_node_oci_inference_service(
+    request: FixtureRequest,
+    unprivileged_client: DynamicClient,
+    multi_node_serving_runtime: ServingRuntime,
+) -> Generator[InferenceService, Any, Any]:
+    resources = {
+        "requests": {
+            "cpu": "1",
+            "memory": "4G",
+        },
+        "limits": {
+            "cpu": "2",
+            "memory": "12G",
+        },
+    }
+
+    worker_resources = {
+        "containers": [
+            {
+                "name": "worker-container",
+                "resources": resources,
+            }
+        ]
+    }
+
+    # NOTE: In KServe v0.15, the autoscaler_mode needs to be updated to "none".
+    with create_isvc(
+        client=unprivileged_client,
+        name=request.param["name"],
+        namespace=multi_node_serving_runtime.namespace,
+        runtime=multi_node_serving_runtime.name,
+        storage_uri="oci://registry.redhat.io/rhelai1/modelcar-granite-8b-code-instruct:1.4",
```

Comment:
add image to constant

Reply:
I extracted. Hopefully, I did it to the right constants location.